pkgs/by-name/aw/aws-signing-helper/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+buildGoModule rec {
+  pname = "aws-signing-helper";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "aws";
+    repo = "rolesanywhere-credential-helper";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-1488l381449b4z3y5zyj121drax1v0ib0xhsmdy1prsmfck1k3xx";
```

Comment:
Apologies, now the format has changed you would need to recalculate this

I just checked and the value should be `sha256-vY8ZJnNV5xt8qxp2sCLYoavcggjS/+LHJysREtCgCJE=`

Reply:
Thanks, fixed